components/flowerFooter/FlowerFooter.tsx

```diff
@@ -77,15 +63,16 @@ const FlowerFooter = ({ sx }: flowerFooterProps) => {
             </Box>
             <Box
               sx={{
-                maxWidth: '600px',
-                p: 10,
-                gap: 4,
-                background: '#9C6E4F',
-                borderRadius: 5,
                 display: 'flex',
+                flex: 1,
                 alignItems: 'center',
                 flexDirection: 'column',
-                textAlign: 'center'
+                textAlign: 'center',
+                maxWidth: 600,
+                p: 10,
+                gap: 4,
+                background: '#9C6E4F',
```

Comment:
**thought:** We might want to start keeping hardcoded colors and other similar styles in the theme to improve consistency, do you think we should add this color to the `defaultTheme`?

Reply:
Yes not a bad idea, i wasn't sure how far we usually would go in situations like this as i imagine considerate cat having a ton of colors by the end of it. But good idea to keep consistent. Will add.

Reply:
Done!